src/Themes.scss

```diff
@@ -286,14 +286,29 @@ body.light-theme {
         background-color: #D5D8DD;
     }
 
-    button.btn.PlayButton, button.btn.StopButton {
+    button.btn.PlayButton {
         svg {
             circle, path.triangle {
                 fill: #3A3F46;
             }
         }
     }
 
+    .StopButton {
+        box-shadow: 0 0 0 0.2rem #3A3F46;
+
+        svg rect {
+            fill: #3A3F46;
+        }
+
+        &:focus {
+            box-shadow:
+                0 0 0 0.2rem #3A3F46,
+                0 0 0 0.4rem #D5D8DD,
+                0 0 0 0.6rem black;
```

Comment:
We should use #1E1E1E for Stop button in Themes.scss too.

Reply:
Done.